pkg/testcommon/testcommon.go

```diff
@@ -67,6 +69,23 @@ func (site *TestSite) Prepare() error {
 		site.Cleanup()
 	}
 
+	// If our test site has a Name: then update the config file to reflect that.
+	if site.Name != "" {
```

Comment:
@rfay Could this possibly be handled by opening the existing config file of the test site and appending the name to it? It seems like that would address this need without having an import cycle between testcommon and ddevapp.